<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>The Photo Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="workspaceView"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="paletteView">
        <item>
         <widget class="QTreeWidget" name="palette">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string notr="true">Palette</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icons/resources/palette-solid.svg</normaloff>:/icons/resources/palette-solid.svg</iconset>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_As">
     <property name="title">
      <string>Save As</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="menuSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuHistory">
     <property name="title">
      <string>History</string>
     </property>
     <addaction name="actionCommit_Changes"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="menuHistory"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFAQs"/>
   </widget>
   <widget class="QMenu" name="menuMultiplayer">
    <property name="title">
     <string>Multiplayer</string>
    </property>
    <addaction name="actionCreate_Room"/>
    <addaction name="actionJoin_Room"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMultiplayer"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/plus-square-regular.svg</normaloff>:/icons/resources/plus-square-regular.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/folder-open-regular.svg</normaloff>:/icons/resources/folder-open-regular.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/save-regular.svg</normaloff>:/icons/resources/save-regular.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/list-ul-solid.svg</normaloff>:/icons/resources/list-ul-solid.svg</iconset>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/history-solid.svg</normaloff>:/icons/resources/history-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/users-solid.svg</normaloff>:/icons/resources/users-solid.svg</iconset>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionFAQs">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/question-circle-regular.svg</normaloff>:/icons/resources/question-circle-regular.svg</iconset>
   </property>
   <property name="text">
    <string>FAQs</string>
   </property>
  </action>
  <action name="actionCreate_Room">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/server-solid.svg</normaloff>:/icons/resources/server-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Create Room</string>
   </property>
  </action>
  <action name="actionJoin_Room">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/network-wired-solid.svg</normaloff>:/icons/resources/network-wired-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Join Room</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/print-solid.svg</normaloff>:/icons/resources/print-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCommit_Changes">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/resources/commit.svg</normaloff>:/icons/resources/commit.svg</iconset>
   </property>
   <property name="text">
    <string>Commit_Changes</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
